generated/command_history.bat

```diff
@@ -1,45 +1,130 @@
 rem This is the list of commands that make invoked in order to build nimi-python. If you want to reproduce the build but don't have GNU Make setup in your system, you can invoke this script.
```

Comment:
Remove this file from Git, we're no longer keeping in source control.
See https://github.com/ni/nimi-python/pull/233

Reply:
Done